tests/lib/azure/vnet.go

```diff
@@ -24,13 +25,21 @@ func NewVnet(f fixture.F) *Vnet {
 
 // AssertExists checks if virtual network exists in the resource group.
 // Fail tests otherwise.
-func (vnet *Vnet) AssertExists(t *testing.T, name string, address string) {
+func (vnet *Vnet) AssertExists(t *testing.T, name string, address, routeTable string) {
```

Comment:
I'm not a huge fan of omitting type on parameters, but it is a personal choice and I don't have a lot of arguments in favor of it besides taste. One time I removed the last argument of a function on a refactoring and got build errors because of the lack of types (the type was only on the last parameter). But it is not very common.

What I don't like here is the lack of consistency, the name parameter have the type, but the address does not have, but they all are strings. It could be:

```
func (vnet *Vnet) AssertExists(t *testing.T, name string, address string, routeTable string)
```
or

```
func (vnet *Vnet) AssertExists(t *testing.T, name, address, routeTable string)
```

But mixing seems really odd.